controller.go

```diff
@@ -22,9 +22,57 @@ import (
 	"net/http"
 	"strconv"
 
+	caoapi "cloud-android-orchestration/api/v1"
+
 	"github.com/gorilla/mux"
 )
 
+type ErrorMsg struct {
+	Error string `json:"error"`
+}
```

Comment:
This should be in api/v1/signalingserver.go or in api/v1/common.go if you want to reuse it for something else. Feel free to rename it if it collides with something.

Reply:
moved to signalingserver.go